docs/content/installation/install_hostdefiner.md

```diff
@@ -0,0 +1,13 @@
+# Installing the host definer
+
+Install the HostDefiner custom resource to enable dynamic host definitions of the CSI driver nodes.
```

Comment:
I would change "to enable dynamic host definition on the storage, by the host definer..."
@oriyarde / @matancarmeli7 thoughts?

Reply:
IMO this just makes the sentence longer, adding only obvious information